.env.local

```
@@ -0,0 +1 @@
+OPENAI_API_KEY=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
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)